turbo.json

```diff
@@ -26,21 +34,38 @@
       ]
     },
     "playroom-start": {
-      "dependsOn": ["build"]
+      "dependsOn": [
+        "@buildo/bento-design-system#build"
+      ],
+      "persistent": true,
+      "cache": false
     },
     "playroom-build": {
-      "dependsOn": ["build"]
+      "dependsOn": [
+        "@buildo/bento-design-system#build"
+      ],
+      "outputs": [
+        "dist"
+      ]
     },
     "website-start": {
       "dependsOn": [
         "^build"
-      ]
+      ],
+      "persistent": true,
+      "cache": false
     },
     "website-build": {
       "dependsOn": [
         "^build"
+      ],
+      "outputs": [
+        "build"
       ]
     },
-    "check-circular-deps": {}
+    "check-circular-deps": {},
+    "wait": {
+      "cache": false
+    }
```

Comment:
do we need this? Isn't it sufficient to define a `wait` script in the project?

Reply:
> Isn't it sufficient to define a `wait` script in the project?

nope. It seems scripts used in the `dependsOn` must be defined in `turbo.json`, otherwise we get a:

> Error preparing engine: cannot find task for ___ROOT___#wait